city/schemas.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import BaseModel
+
+
+class CityCreate(BaseModel):
+    name: str
+    additional_info: str
```

Comment:
The class `Temperature` is defined here, but this file is expected to define the `City` schema according to the task description. Ensure that you define a `City` Pydantic model with fields `id`, `name`, and `additional_info` as required by the task.